app/src/main/java/mozilla/lockbox/view/ItemDetailFragment.kt

```diff
@@ -69,6 +69,16 @@ class ItemDetailFragment : BackableFragment(), ItemDetailView {
             updatePasswordVisibility(value)
         }
 
+    override fun onPause() {
```

Comment:
can you add these calls to the base `Fragment` class? or ensure that they don't already exist there?

Reply:
yes! they didn't exist there before. Is it always best to put those overrides in the `Fragment`?

Reply:
I think you're fine leaving the `onDestroyView` instead of `onDestroy` in there... and yeah, the idea with the base `Fragment` class is that individual `Fragment`s don't need to call any additional lifecycle actions (remembering to call `onViewReady` every time etc), you can just implement them in the `Presenter` as needed.